OpenRA.Mods.RA/Traits/GpsRemoveFrozenActor.cs

```diff
@@ -35,21 +34,31 @@ public GpsRemoveFrozenActor(Actor self, GpsRemoveFrozenActorInfo info)
 			watchers = self.World.ActorsWithTrait<GpsWatcher>().Select(w => w.Trait).ToArray();
 		}
 
-		public bool RemoveActor(Actor self, Player owner)
+		public void Tick(Actor self)
```

Comment:
You can't guarantee this to be called after an actor is killed.  You should use INotifyRemovedFromWorld or similar instead.